### PR TITLE
Soarian R4 capabilityStatement Documentation

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
----
Soarian R4 capabilityStatement Documentation
PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
CS_1
CS_2
CS_3
CS_4
CS_5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/foundation/conformance/capabilitystatement.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<%= headers status: 200 %>
+<%= json(:SOARIAN_R4_RESPONSE_PROVIDER_AND_SYSTEM_PERSONA) %>
+
+
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply content/soarian/r4/foundation/conformance/capabilitystatement.md]: Removed extra lines

File: layouts/soarian/r4/foundation/conformance.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Resources in the Conformance category are used to represent metadata about the datatypes, resources, and API features of the FHIR<sup>®</sup> specification and can be used to create derived specifications.</p>
+
+<p>Select <a href="https://hl7.org/fhir/r4/conformance-module.html">here</a> to see the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>
+
+<h3 id="implemented-resources">Implemented Resources</h3>
```
[the remark this answers is not in the document]
Comment (DeepakGupta93) [reply layouts/soarian/r4/foundation/conformance.html]: Added a empty line....

**Comment by DeepakGupta93** [comment]
> Apart from these, Millennium Capability resource has a nice way of retrieving the results Live. If possible try to include that along with the example response.

This comment we will be addressing once we get a public sandbox ready and working... I will work with Sridhar and have it implemented here.

**Review by ashu-bdvt — APPROVED** [review]

**Review by slvsridhar — APPROVED** [review]

**Comment by DeepakGupta93** [comment]
Closing this PR as Shiladitya is working on same resource with latest changes.
https://github.com/cerner/fhir.cerner.com/pull/823

**Pull request closed without merging**